### ISSUE TITLE
Kovan ETH testing deposit of funds

### ISSUE THREAD

**Issue by muhammaddadu** [openware/opendax#197]
Hi,

I am running Kovan ETH and have created a [wallet](https://kovan.etherscan.io/address/0xd6c208973cf8bdd938c4d3925e294327f03f0c30) inside my instance of opendax.

I have transfered 1ETH to the address as seen in etherscan. I can see from the logs that the blocks match up but I cannot see the funds in my account.

<img width="531" alt="image" src="https://user-images.githubusercontent.com/5151809/89179021-c7a47800-d586-11ea-84fc-2522d43bc2fb.png">
<img width="821" alt="image" src="https://user-images.githubusercontent.com/5151809/89179043-d559fd80-d586-11ea-9fa1-cce93ef6556b.png">

**Comment by Galaxy-Traveler** [comment]
You created depozit wallet?

**Comment by muhammaddadu** [comment]
Yes, deposit wallet created and connected to correct blockchain.

**Comment by Galaxy-Traveler** [comment]
Your wallet is local or offline.
Local:
curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"personal_newAccount","params":["pretty_safe_passphrase"],"id":42}' localhost:8545
------------
curl --data '{"method":"parity_testPassword","params":["*account address*","*account_passphrase*"],"id":1,"jsonrpc":"2.0"}' -H "Content-Type: application/json" -X POST localhost:8545

**Comment by devshaaran** [comment]
How to use an online wallet?

**Comment by alinetskyi** [comment]
> Yes, deposit wallet created and connected to correct blockchain.

Have you also created a blockchain in the admin panel and does the block number match up in there ?

**Comment by talhadar90** [comment]
Any fix to this

**Comment by hsklia** [comment]
same issue
